cmd/pack.js

```diff
@@ -24,8 +24,21 @@ export default async function pack (filePath, opts = {}) {
   const rest = opts._ ?? []
   const paths = checkPathsExist([filePath, ...rest].filter(Boolean))
   const hidden = !!opts.hidden
+  let fullOutputPath
+  if (opts.output) {
+    fullOutputPath = path.resolve(opts.output)
+  }
```

Comment:
```suggestion
  const fullOutputPath = opts.output !== null ? path.resolve(opts.output) : null;
```

- stricter check
- remove unnecessary modifiableness

Reply:
I'm not convinced this will work. It's worth checking but I don't _think_ `opts.output` is ever `null`. It might be `undefined` or the empty string.